source/computer/operating-system.rst

```diff
@@ -1,2 +1,51 @@
-操作系统
-========
+选择操作系统
+============
+
+个人电脑有三种常见的操作系统：Microsoft Windows、macOS 和 Linux。
+从市场占有率上看，Microsoft Windows 用户数目远远多于 macOS 和 Linux 用户。
+然而，大多数地震学科研工作者都选择 Linux 或 macOS 作为日常研究的主要操作系统。
+主要原因有如下几点：
+
+1. Linux 和 macOS 上可以更容易的获取和使用编程所需的开发工具
+2. 大多数地震学相关的程序/代码都是在 Linux 上编写的，且仅在 Linux 和 macOS 下可以使用
+3. Linux 和 macOS 自带了可以对数据做基本处理的命令行工具
+4. 高性能计算服务器几乎都只使用 Linux 系统
+
+对于初学者，我们\ **建议使用 Linux 或 macOS 系统**\ ，除非你非常确定自己的科研
+工作并不需要用到别人提供的代码，或者你有足够的知识和能力在 Windows 系统下
+编译和使用别人的代码。
+
+macOS
+-----
+
+Mac 笔记本或台式机（如 MacBook Air、MacBook Pro、iMac）自带的 macOS 系统
+可以满足科研工作的大部分需求。对于拥有 macOS 系统的用户，建议直接使用 macOS 系统。
+可以参考《\ :doc:`/computer/macos-setup`\ 》对 macOS 系统进行进一步配置，以满足
+科研工作的需求。
+
+Linux
+-----
+
+Linux 有很多不同的发行版，所有 Linux 发行版都可以免费获取与安装。
+比较流行的发行版有
+`Ubuntu <https://ubuntu.com/>`__\ 、\
+`Debian <https://www.debian.org/>`__\ 、\
+`Fedora <https://getfedora.org/>`__\ 、\
+`CentOS <https://www.centos.org/>`__ 等。
+读者可以根据自己的喜好自行选择合适的 Linux 发行版。
+
```

Comment:
```suggestion

.. note::

   Red Hat 在 2020 年 12 月宣布将停止 CentOS Linux 项目。CentOS Linux 8 将在
   2021 年末停止更新。同时，考虑到广大用户对 CentOS Linux 7 的使用需求，
   Red Hat 将继续更新和维护其至 2024 年 6 月 30 日。详情请阅读
   `CentOS 通告 <https://lists.centos.org/pipermail/centos-announce/2020-December/048208.html>`__。

```

Reply:
Do we really want to mention centos 8? centos stream is still promising although i may not use it anymore.

Reply:
I think we may better let babies kown the announce. you may add that centos linux 8 is replaced by centos stream. We can delete the note when we have a decision about what linux distribution we will use